app/src/main/java/com/github/se/polyfit/ui/components/AppTitleTopBar.kt

```diff
@@ -0,0 +1,35 @@
+package com.github.se.polyfit.ui.components
+
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.material3.TopAppBar
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.text.font.FontWeight
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun AppTitle(titleText: String = "Polyfit", alignment: Alignment = Alignment.TopCenter) {
+  TopAppBar(
+      modifier = Modifier.testTag("MainTopBar"),
+      title = {
+        Box(contentAlignment = Alignment.Center, modifier = Modifier.fillMaxWidth()) {
+          GradientBox(outerModifier = Modifier.fillMaxWidth(0.8f)) {
+            // align box at the center
+            Box(contentAlignment = Alignment.Center, modifier = Modifier.fillMaxWidth()) {
+              Text(
+                  text = titleText,
+                  style = MaterialTheme.typography.titleLarge,
+                  color = MaterialTheme.colorScheme.primary,
```

Comment:
color should be onSurface as the TopAppBar will be display with surface color or it depends the color of the Box. I cannot see what's look like in the review, but it could lead to color problem

Reply:
the box is transparent so I guess it's alright to let the text color as primary